src/coordinates.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * Parses Coordinates for pages where the default Wiki Infobox Parser fails.
+ * @param  {Object} infoboxData - raw data object from Wiki Infobox Parser
+ * @return {Object} - formatted object containing coordinates, or null object if none.
+ */
+export function parseCoordinates(infoboxData) {
+	if(infoboxData.coordinates) {
+		// inline coordinate format
+		return parseInfoboxCoords(infoboxData.coordinates);
+	}
+	else if(infoboxData.latd && infoboxData.longd) {
```

Comment:
You shouldn't need this else/if 

You can just use the two conditionals... like this..

```js
if(infoboxData.coordinates) {
  return parseInfoboxCoords(infoboxData.coordinates);
}
if(infoboxData.latd && infoboxData.longd) {
  return parseDeprecatedCoords(infoboxData.latd,infoboxData.longd);
}
return {
  lat: null,
  lon: null,
  error: 'No coordinates on page.'
};
```